src/store/Modules/Connections.ts

```diff
@@ -85,10 +85,11 @@ export default class Connections extends VuexModule {
       this.context.commit("addGlobalConnections", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("addGlobalConnections", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.